CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Version History
 ===============
 
+v7.0.0
+------
+
+* Update core dependencies `<https://github.com/lsst-ts/LOVE-manager/pull/270>`_
```

Comment:
Looks like more happened, like remove Authlist support. Not sure if you want to mention that here as well?